ocpp-wamp/src/main/kotlin/com/izivia/ocpp/wamp/server/OcppWampServer.kt

```diff
@@ -62,8 +62,9 @@ interface OcppWampServer {
             path: String = "ws",
             timeoutInMs: Long = 60_000,
             onWsConnectHandler: (CSOcppId, WampMessageMetaHeaders) -> Unit = { _, _ -> },
+            onWsReconnectHandler: (CSOcppId, WampMessageMetaHeaders) -> Unit = { _, _ -> },
```

Comment:
this breaks api compatiblity with non named arguments, but considering limited use of the api, I think it's ok. Should be published as a 1.1.0 to warn about that